SmartDeviceLink/SDLVersion.h

```diff
@@ -31,6 +31,7 @@ NS_ASSUME_NONNULL_BEGIN
 - (BOOL)isLessThanVersion:(SDLVersion *)otherVersion;
 - (BOOL)isEqualToVersion:(SDLVersion *)otherVersion;
 - (BOOL)isGreaterThanVersion:(SDLVersion *)otherVersion;
+- (BOOL)isGreaterThanOrEqualVersion:(SDLVersion *)otherVersion;
```

Comment:
Change to `isGreaterThanOrEqualToVersion`

May as well also add `isLessThanOrEqualToVersion`

Reply:
Added